### ISSUE TITLE
LN reader error

### ISSUE THREAD

**Issue by Kleinmoretti247** [LNReader/lnreader#1292]
### Steps to reproduce

When I want to download or update a chapter in LNreader, it always gives an error and always exits the application. also there is always the info "LNreader has stopped". I don't know what that means, there's no problem with the plugins. can any of you help

### Expected behavior

can't download chapters

### Actual behavior

can't download chapters

### LNReader version

2.0.0

### Android version

10.2.3

### Device

MIUI global 10.2.3

### Other details

__When I want to download or update a chapter in LNreader, it always gives an error and always exits the application. also there is always the info "LNreader has stopped". I don't know what that means, there's no problem with the plugins. can any of you help

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with a source, I should be opening an issue in the [sources repository](https://github.com/lnreader/lnreader-sources/issues/new/choose).
- [X] I have updated the app to version **[2.0.0](https://github.com/lnreader/lnreader/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by Kleinmoretti247** [comment]
> Can you try making a Google drive backup and deleting the app?

no, I don't know why, when I do a backup on the drive it will immediately exit the application. What is the problem?

**Comment by ilonachan** [comment]
Does this problem happen with all sources, or just one/multiple specific ones? Giving a novel link as an example would be helpful.

**Comment by Palloxin** [comment]
@rajarsheechatterjee unconclusional issue, prob to close
